pavics_datacatalog/wps_processes/wps_pavicrawler.py

```diff
@@ -6,6 +6,10 @@
 
 from pavics import catalog
 
+#TODO: I suggest to move that configuration stuff into a config.py file which create an `env` variable that can be used elsewhere. Talk to Fred about the best way to go about it.
+#TODO: Change the name of the process. It not only crawls but also update the SOLR server.
+# How about making the THREDDS and SOLR servers explicit in the call, ie as inputs? I think better yet would be the crawling process and the update process.
+
```

Comment:
This "configuration stuff" is currently set in catalog.cfg, and set as environment variables in the Dockerfile. This was necessary before the DataCatalog was also a python package. I guess it can be changed to your suggestion now.